File: assignment/Spicysnack/calculate_average.py
scoreOne = float(input("Enter first score: "))
scoreTwo = float(input("Enter second score: "))
scoreThree = float(input("Enter third score: "))

sum_scores = scoreOne + scoreTwo + scoreThree
average = sum_scores / 3

print("Average is:", average)

if 90 <= average <= 100:
    print("Grade is A")
elif average >= 80:
    print("Grade is B")
elif average >= 70:
    print("Grade is C")
elif average >= 60:
    print("Grade is D")
else:
    print("Grade is F")

File: assignment/javaPython/CalculateAverage.java
import java.util.Scanner;
public class CalculateAverage{
public static void main(String[]args){

Scanner input = new Scanner(System.in);
System.out.print(" Enter first score: ");
double scoreOne = input.nextDouble();

System.out.print(" Enter Second score: ");
double scoreTwo = input.nextDouble();

System.out.print(" Enter Third score: ");
double scoreThree = input.nextDouble();

double sum = scoreOne + scoreTwo + scoreThree;

double average = sum / 3;

if (average >= 90 && average <= 100){
System.out.println("Score is A");
}

 else if (average >= 80){
System.out.println("print B");
}
else if (average >= 70){
 System.out.println("print C" );
}
else if (average >= 60){
 System.out.println("print D" );
}
else{
System.out.println("print F" );
}
}
}
